pycroft/helpers/i18n.py

```diff
@@ -337,7 +337,7 @@ class Message(object):
     @classmethod
     def from_json(cls, json_string):
         try:
-            obj = json.loads(json_string, encoding='utf8')
+            obj = json.loads(json_string)
```

Comment:
How is this related to the PR? In both cases this should be the py3.7  stdlib `json`-module, so there should'nt be any breaking API changes

Reply:
As written in the commit description, the `encoding` parameter has been deprecated since 3.1 and was removed with 3.9.

Reply:
Thanks for the clarification, didn't see the corresponding commit message. Fine by me